parachain/src/command.rs

```diff
@@ -319,17 +325,23 @@ pub fn run() -> Result<()> {
             let _ = builder.init();
 
             let raw_wasm_blob = extract_genesis_wasm(&cli.load_spec(&params.chain.clone().unwrap_or_default())?)?;
-            let output_buf = if params.raw {
-                raw_wasm_blob
-            } else {
-                format!("0x{:?}", HexDisplay::from(&raw_wasm_blob)).into_bytes()
-            };
+            write_to_file_or_stdout(params.raw, &params.output, raw_wasm_blob)?;
 
-            if let Some(output) = &params.output {
-                std::fs::write(output, output_buf)?;
-            } else {
-                std::io::stdout().write_all(&output_buf)?;
-            }
+            Ok(())
+        }
+        Some(Subcommand::ExportMetadata(params)) => {
+            let mut ext = frame_support::BasicExternalities::default();
+            sc_executor::with_externalities_safe(&mut ext, move || {
+                let raw_meta_blob = match params.runtime {
+                    RuntimeName::Interlay => interlay_runtime::Runtime::metadata().into(),
```

Comment:
Would it be possible to have a `chain` argument rather than a `runtime` argument? That'd be more in line with the other commands, and also I think the chain selection impacts the constants that are part of the metadata

Reply:
The chain selection does not impact the constants that are part of the metadata AFAIK.

Reply:
Hmm I think you are right - I guess the only difference between chains of the same runtime is the genesis config, which doesn't change constants